pkg/apis/pipeline/v1beta1/taskrun_validation_test.go

```diff
@@ -63,7 +63,7 @@ func TestTaskRun_Invalidate(t *testing.T) {
 			Message: `non-existent variable in "$(params.task-words[*])"`,
 			Paths:   []string{"spec.steps[0].args[0]"},
 		},
-		wc: config.EnableAlphaAPIFields,
+		wc: config.EnableBetaAPIFields,
```

Comment:
we do not need this check in v1beta1 - see https://github.com/tektoncd/pipeline/pull/5628#discussion_r993724518